config/locales/diaspora/en.yml

```diff
@@ -1106,6 +1106,8 @@ en:
   shared:
     aspect_dropdown:
       add_to_aspect: "Add contact"
+      mobile_row_checked: "✓ %{name} (remove)"
+      mobile_row_unchecked: "– %{name} (add)"
```

Comment:
Have already checked. This works on desktop browsers, but in mobile browsers the dropdown is rendered as popup-modal-dialog on your phone and that dialog does not respect any css or anything other than "this is the utf-8 text of the option". Not even background-color or anything. So ::before won't work.